src/a1atscript/ng2Directives/PropertiesBuilder.js

```diff
@@ -2,23 +2,21 @@ import BindBuilder from "./BindBuilder.js"
 
 const BIND_PREFIX = "_=_";
 const STRING_PREFIX = "_@_";
-
-const USING_DATA_BINDING = 1;
-const USING_RAW_STRING = 2;
+const BINDING = BIND_PREFIX;
+const RAW_STRING = STRING_PREFIX;
```

Comment:
I guess it was a semantics thing. I considered it. I'll change it along with a future pr.